src/cplscheme/tests/DummyCouplingScheme.hpp

```diff
@@ -37,6 +37,14 @@ class DummyCouplingScheme : public CouplingScheme {
       double startTime,
       int    startTimesteps) override final;
 
+  /**
+   * @brief Not implemented.
+   */
+  void receiveResultOfFirstAdvance()
```

Comment:
```suggestion
  void receiveResultOfFirstAdvance() override final
```
Please double-check this suggestion, don't know the C++ rules here by heart.